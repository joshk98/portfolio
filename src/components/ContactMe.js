import React from "react";
import "../styles/contactme.css";

function ContactMe() {
  return (
    <div className="contactme-container">
      Let us stay in touch!
      <div className="contactme-container-links">
        <a
          href="https://linkedin.com/in/joshua-kuhn"
          target="_blank"
          rel="noopener noreferrer"
        >
          <img
            src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/linkedin/linkedin-original.svg"
            alt="linkedin"
          />
        </a>
        <a
          href="https://github.com/joshk98"
          target="_blank"
          rel="noopener noreferrer"
        >
          <img
            src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/github/github-original.svg"
            alt="github"
          />
        </a>
        <a href="mailto:joshkkuhn@gmail.com">
          <img
            src="https://www.svgrepo.com/show/303161/gmail-icon-logo.svg"
            alt="email"
          />
        </a>
      </div>
    </div>
  );
}

export default ContactMe;
