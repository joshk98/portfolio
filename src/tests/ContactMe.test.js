import React from "react";
import { render } from "@testing-library/react";
import ContactMe from "../components/ContactMe";

describe("ContactMe", () => {
  it("renders correctly", () => {
    const { asFragment } = render(<ContactMe />);

    expect(asFragment()).toMatchSnapshot();
  });
});
