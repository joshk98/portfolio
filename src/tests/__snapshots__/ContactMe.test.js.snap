// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactMe renders correctly 1`] = `
<DocumentFragment>
  <div
    class="contactme-container"
    id="contactme-section"
  >
    <h2>
      Let's keep in touch! :)
    </h2>
    <div
      class="contactme-container-links"
    >
      <a
        href="https://linkedin.com/in/joshua-kuhn"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="linkedin"
          src="icon-linkedin.png"
        />
      </a>
      <a
        href="https://github.com/joshk98"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="github"
          src="icon-github.png"
        />
      </a>
      <a
        href="mailto:joshkkuhn@gmail.com"
      >
        <img
          alt="email"
          src="icon-email.png"
        />
      </a>
    </div>
  </div>
</DocumentFragment>
`;
